src/main/java/mil/dds/anet/beans/Task.java

```diff
@@ -58,9 +58,9 @@
   @GraphQLInputField
   TaskStatus status;
   // annotated below
-  private ForeignObjectHolder<Organization> responsibleOrg = new ForeignObjectHolder<>();
```

Comment:
Don't forget to update the tests (`src/test/java/mil/dds/anet/test/resources/TaskResourceTest.java`, `src/test/java/mil/dds/anet/test/TestData.java`, `src/test/resources/graphQLTests/{report-all-recents.gql,task-recents.gql,report.gql, task-recents-maxResults.gql}` and the email templates (`src/main/resources/emails/emailReport.ftlh`).